src/overlays/actors/ovl_En_Siofuki/z_en_siofuki.h

```diff
@@ -4,11 +4,33 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum {
+    /* 0x00 */ RAISING,
+    /* 0x01 */ LOWERING
```

Comment:
Enum values should usually have a prefix to avoid any conflicts and make their purpose obvious, especially if they are exposed in a header for public external usage.
```suggestion
    /* 0x00 */ EN_SIOFUKI_RAISING,
    /* 0x01 */ EN_SIOFUKI_LOWERING
```